account_payment_order/migrations/18.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env, "account_payment_order", "18.0.1.0.0/noupdate_changes.xml"
```

Comment:
```suggestion
        env, "account_payment_order", "migrations/18.0.1.0.0/noupdate_changes.xml"
```
When I tried to update the module, the migration script fails due to the fact that the path written in the script can't find the file

Reply:
Solved, thanks!